include/industry_standard/spdm.h

```diff
@@ -14,6 +14,13 @@
 
 #pragma pack(1)
 
+#define SPDM_MAX_VERSION_COUNT 5

```

Comment:
I think spdm does not define MAX count.
But if we count SPDM 1.2, we need redefine it to 3.

We can handle that later, in another patch.

Reply:
According to https://github.com/DMTF/Security-TF/issues/1690 it can be as large as 256 if the Responder is allowed to duplicate entries. :(